### ISSUE TITLE
Is there a way to disable codesandbox?

### ISSUE THREAD

**Issue by weslleyaraujo** [pedronauck/docz#386]
## Question

For the project that I am working (private repo etc) I would like not to have codesandbox intergrated. It would be handy if we could disable that from doczrc config.

Cheers! 🎉

**Comment by wsmd** [comment]
I started working on this and some made good progress, but I've hit a wall, and I'd appreciate any guidance (I'm new the project!)

The first naive approach I considered at first was to hide the codesandbox button based on the theme configuration, but I noticed that the playground component, BasePlayground, passes the codesandbox parameters down to the theme itself. This approach won't be ideal since all of the logic to generate the codesandbox files would still happen before it even gets passed down to theme!

The logic for generating the codesandbox info is done via the rehype plugin, `rehype-docz`, at startup time as soon as the `dev` command starts.

I managed to pass the configuration down to the rehype plugin itself, and I was able to conditionally get the codesandbox info...

![untitled 2018-10-10 01_09_16](https://user-images.githubusercontent.com/2100222/46714379-20c05080-cc29-11e8-821d-310a65a1c2eb.gif)

The problem that encountered is caching. The codesandbox info is tied to each document, so when toggling this option on and off, a new version of the document has to be created, by triggering a change for example. It won't be recalculated automatically for all documents.

I'm happy to open a PR if what I have is a good start.
